examples/test_examples.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+pip install fvcore iopath scikit-learn pandas matplotlib seaborn dgl dglgo -f https://data.dgl.ai/wheels/repo.html
+ 
+python classification.py
+
+python regression.py
+
+python vcl.py --dataset mnist --inference ml --test # can't download and run cifar succesfully
+python vcl.py --dataset mnist --inference mean-field --test
+
+python gnn.py --num-epochs 100
+python resnet.py --inference ml --num-epochs 10
+python resnet.py --inference mean-field --num-epochs 10
+python resnet.py --inference map --num-epochs 10
+
+# Adding a test for one of the last-laye rones would be nice, but not sure how to handle the shipping of pre-trained weights
+# python resnet.py --inference last-layer-mean-field --num-epochs 10
```

Comment:
See comment in resnet.py file

Reply:
We could take the initial random ones in the script since it's just for testing that the script doesn't crash.